_episodes/17-scope.md

```diff
@@ -106,4 +106,12 @@ NameError: name 'temperature' is not defined
 > KeyError: 'Friday'
 > ~~~
 > {: .error}
+> > ## Solution
+> > 1. Three levels.
+> > 2. `errors_02.py`
+> > 3. `print_message`
+> > 4. Line 11
+> > 5. `KeyError`
```

Comment:
Perhaps we can also explain what a KeyError is or link to https://docs.python.org/3/library/exceptions.html#KeyError